src/main/java/uk/gov/hmcts/juror/api/moj/utils/NumberUtils.java

```diff
@@ -19,4 +19,8 @@ public static int unboxIntegerValues(Integer integer) {
             :
                 integer;
     }
+
+    public static int calculatePercentage(long numerator, long denominator) {
+        return denominator == 0 ? 0 : (int) Math.round((numerator * 100.0) / denominator);
+    }
```

Comment:
Percentage should return a double instead of an integer.
BE should be returning an accurate percentage with decimals (With FE round if it does not want them)